api/_lib/FanoutGraphqlExpressServer.ts

```diff
@@ -265,9 +266,3 @@ const main = async () => {
       console.log(`🚀 Subscriptions ready at ${subscriptionsUrl}`);
     });
 };
-
-if (require.main === module) {
```

Comment:
Why remove this? Now the file won't do anything when executed.

Reply:
(Which can be useful when, e.g. I'm editing this file in an IDE liek VS Code and you want to run/debug the file by pressing F5)

Reply:
I'll need to double check this change. Perhaps it can remain the same.

Reply:
I reverted, thanks